installer/roles/ansible-service-broker-setup/tasks/main.yml

```diff
@@ -28,6 +28,24 @@
     - apb_sync is defined
     - apb_sync
 
+- name: Check If OpenShift Ansible Broker Is Installed
+  shell: oc get namespace "{{ ansible_service_broker_namespace }}"
+  register: asb_namespace
+  failed_when: false
+  no_log: True
+
+- name: Ensure OpenShift Ansible Broker Is Deprovisioned
+  shell: >
```

Comment:
So this checks if the asb namespace is present and if not, it removes the remaining asb artifacts?

Reply:
Yes exactly. This was the list of artifacts with the error `already exists` when trying to re-provision ASB after it had been deleted